core/ic/src/tera/src/main.rs

```diff
@@ -96,7 +96,7 @@ mod tests {
 
         assert_eq!(slice.len(), 28);
 
-        let p = Principal::from_nat(n);
+        let p = Principal::from_nat(n).unwrap();
```

Comment:
if we return a result, shouldn't we handle it?

Reply:
yes, but this is just a test, I expect unwrap not to panic